recipe/meta.yaml

```diff
@@ -16,7 +16,7 @@ source:
 
 build:
   number: {{ build }}
-  skip: true  # [(win and py==310) or (cuda_compiler_version in (undefined, 'None') and (linux64 or win)) or (cuda_compiler_version != '10.2')]
+  skip: true  # [(win and py==310) or (cuda_compiler_version in (undefined, 'None') and (linux64 or win))]
   script_env:
     - CONFIG
```

Comment:
What is this used for?

Reply:
I am not sure, I had been using OpenMM's feedstock settings as a starting point, and I think I just included this because OpenMM included it as well.

Reply:
Ok I think [OpenMM is using this for some sort of testing check]( https://github.com/conda-forge/openmm-feedstock/commit/7f612876acd4576fd76698b1afe387871a1dc8dd ) that we don't have here. Let's go ahead and drop this